package.json

```diff
@@ -11,7 +11,9 @@
   "author": "Eduardo Olivares",
   "license": "MIT",
   "dependencies": {
+    "awilix": "^4.3.1",
     "axios": "^0.21.1",
+    "coingecko-api": "^1.0.10",
```

Comment:
Good idea to use the coingecko-api package as opposed to making raw api requests